styles/stylesheet.css

```diff
@@ -0,0 +1,458 @@
+    @media print {
+        body img {
+          display: none;
+       }
+    }
+    @media screen and (min-width: 200px) {
+        .homeimage {
+            height: 400px;
+            object-fit: contain;
+        }
+    }
+    @media screen and (min-width: 200px) {
+        .blogimage2 {
+          height: 200px;
+          object-fit: contain;
+          padding-top: 30px;
+        }
+    }
+    @media screen and (min-width: 200px) {
+        img {
+            display: inline;
+            width: 100%;
+        }
+    }
+    @media screen and (max-width: 768px) {
+        .previewblog {
+            display: none;
+        }
+    }
+    @media screen and (max-width: 768px) {
+        .grid-container {
+            grid-template-columns: 1fr 1fr;
+        }
+    }
+    @media screen and (max-width: 576px) {
+        .grid-container {
+            grid-template-columns: 1fr;
+        }
+    }
+    @media screen and (max-width: 576px) {
+        h1 {
+            font-size: 50%;
+        }
+    }
+    @media screen and (min-width: 576px) {
```

Comment:
You can combine max-width and min-width onto the same query. As an example:

`@media only screen and (min-width:576) and (max-width:768)`

Reply:
Thanks! I tired to do this, but I struggled with the syntax. How do you code a combine max-width and min-width query if the css within the two separate queries are different?